dev/cognitiveservices/data-plane/Language/analyzetext.json

```diff
@@ -1651,6 +1659,62 @@
       },
       "description": "Represents the confidence scores across all sentiment classes: positive and negative."
     },
+    "CustomSentimentAnalysisTaskParameters": {
+      "type": "object",
+      "description": "Supported parameters for a Custom Sentiment Analysis task.",
+      "properties": {
```

Comment:
Will CustomSentimentAnalysis support opinion mining (now or in the future)?

Reply:
not in the near future. That's why we don't introduce the om parameter here